src/main/java/net/pms/util/SubtitleUtils.java

```diff
@@ -639,7 +639,7 @@ public static InputStream removeSubRipTags(File file) throws IOException {
 		BufferedReader input = FileUtil.bufferedReaderWithCorrectCharset(file);
 		ByteArrayOutputStream os = new ByteArrayOutputStream();
 		Writer writer = new OutputStreamWriter(os, Charset.forName(CHARSET_UTF_8));
-		Pattern pattern = Pattern.compile("\\</?(?:b|i|s|u|font[^\\>]*)\\>|\\{\\\\[^\\}]+\\}|\\\\h|\\\\N");
+		Pattern pattern = Pattern.compile("\\</?(?:b|i|s|u|font[^\\>]*)\\>|\\{\\\\.*?}|\\\\h|\\\\N");
```

Comment:
@Nadahar maybe we can test which of the approach is faster to distinguish between them. 😄